ML/config_Training_LctopK0s_13TeV_pt_4_6.yml

```diff
@@ -0,0 +1,154 @@
+input: # files to use, set FD to null for binary classification
```

Comment:
Is this file needed since you already have the one with 4-50 pt interval?

Reply:
No, I can remove it.